tests/test_brown.py

```diff
@@ -111,116 +201,99 @@ def test_abstract(record):
 
+
 def test_thesis(record):
     """Test thesis year."""
     assert record["thesis"]
     assert record["thesis"]["date"] == "2011"
 
+
 def test_urls(record):
     """Test urls."""
+    url = (
+        "https://repository.library.brown.edu/studio/item/bdr:11303/"
+    )
     assert record["urls"]
-    assert record["urls"][0]["value"] == "brown/test_splash.html"
-
-
-@pytest.fixture
-def parsed_node_no_splash():
-    """Return a parse call to a record without spalsh page url."""
-    spider = brown_spider.BrownSpider()
-    body = """
-    {
-    "items": {
-        "docs": [
-            {
-                "json_uri": "https://repository.library.brown.edu/api/pub/items/bdr:11303/"
-
-            }
-        ]
-    }
-    }
-    """
+    assert record["urls"][0]["value"] == url
 
-    response = fake_response_from_string(body)
-    jsonresponse = json.loads(response.body_as_unicode())
-    jsonrecord = jsonresponse["items"]["docs"][0]
-    response.meta["jsonrecord"] = jsonrecord
-
-    return spider.parse(response).next()
 
-def test_no_splash(parsed_node_no_splash):
-    """Test if parsing a record without splash url results directly in item building.
+def test_no_splash(record_without_splash):
+    """Test if parsing a record without splash url results in a HEPRecord."""
+    assert record_without_splash
+    assert isinstance(record_without_splash, hepcrawl.items.HEPRecord)
+    assert "urls" not in record_without_splash
 
-    Normally `parse` should result in a get request. Here parsed_node_nolink
-    should be final HEPRecord.
-    """
-    assert parsed_node_no_splash
-    assert isinstance(parsed_node_no_splash, hepcrawl.items.HEPRecord)
 
-@pytest.fixture
-def no_year_no_author():
-    """Test behaviour when no year given in thesis info line."""
-    spider = brown_spider.BrownSpider()
-    body = """
-    <html>
-        <div class="panel-body">
-            <dl class="">
-                <dt>Notes</dt>
-                <dd>Thesis (Ph.D. -- Brown University</dd>
-            </dl>
-        </div>
-    </html>
+def test_no_pdf_in_metadata(record_without_pdf):
+    """Test scraping the pdf file."""
+    link = "https://repository.library.brown.edu/studio/item/bdr:11303/PDF/"
 
-    """
-    return fake_response_from_string(body)
+    assert "file_urls" in record_without_pdf
+    assert record_without_pdf["file_urls"][0] == link
 
 
-def test_no_year_in_thesis(no_year_no_author):
+def test_no_year_author_in_thesis(record_no_year_no_author):
     """Test that there is no year."""
-    spider = brown_spider.BrownSpider()
-    year = spider._get_phd_year(no_year_no_author)
+    assert record_no_year_no_author
+    assert "date_published" not in record_no_year_no_author
+    assert "authors" not in record_no_year_no_author
 
-    assert not year
 
-def test_no_author_in_thesis(no_year_no_author):
-    """Test that there are no authors."""
+def test_record_after_pipeline(record, patch_datetime_now):
```

Comment:
So I'll just make this a fixture and return the `final_record` and use it everywhere in place or `record`?